examples/framework/preferences/minimalEditor/js/minEditor.js

```diff
@@ -0,0 +1,131 @@
+/*
+Copyright 2015 OCAD University
+
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one these
+Licenses.
+
+You may obtain a copy of the ECL 2.0 License and BSD License at
+https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
+*/
+
+/* global fluid */
+
+var minEditor = minEditor || {};
+
+(function () {
+    "use strict";
+
+    /**
+     * Primary Schema
+     * This schema defines the preference(s) edited by this preferences editor:
+     * their names, types, default values, etc.
+     */
+    minEditor.primarySchema = {
+        // the string "minEditor.autoPilot" is the 'name' of the preference
+        "minEditor.autoPilot": {
+            type: "boolean",
+            "default": false
+        }
+    };
```

Comment:
Are you sure this doesn't need to be a component. I wonder if it is just working by chance. Primary schemas should also work without actually having to supply them directly to the builder I believe. They are also sourced from the environment. Assuming this hasn't changed since the big pull request.